<launch>
  <node name="height_map_generator" pkg="height_mapping" type="height_map_generator_node" output="screen">
    <param name="data_directory" value="$(find height_mapping)/maps/" />
    <param name="publish_rate" value="1.0" />
    <param name="map_resolution" value="0.1" />
    <param name="min_height" value="-1.0" />
    <param name="max_height" value="2.0" />
    <param name="num_scans_to_aggregate" value="100" />
    <param name="frame_id" value="map" />
  </node>

  <!-- Launch RViz with configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find height_mapping)/launch/rviz/height_map_generator.rviz" />
</launch> 